src/main/java/org/opensearch/sdk/BaseExtension.java

```diff
@@ -16,7 +16,7 @@
 /**
  * An abstract class that simplifies extension initialization and provides an instance of the runner.
  */
-public abstract class BaseExtension implements Extension, ActionExtension {
+public abstract class BaseExtension implements Extension, ActionExtension, ScriptExtension {
```

Comment:
Don't add this here.  In fact, we actually need to remove `ActionExtension` but will do that elsewhere.